packages/db/tests/translate/test-granules.js

```diff
@@ -0,0 +1,97 @@
+const test = require('ava');
+const sinon = require('sinon');
+const cryptoRandomString = require('crypto-random-string');
+
+const { removeNilProperties } = require('@cumulus/common/util');
+const getRecordCumulusId = require('../../dist/database');
+const { translateApiGranuleToPostgresGranule } = require('../../dist/translate/granules');
+
+test('translateApiGranuleToPostgresGranule converts API execution to Postgres', async (t) => {
+  const collectionCumulusId = 1;
+  const providerCumulusId = 2;
+  const executionCumulusId = 3;
+  const pdrCumulusId = 4;
+  const getRecordCumulusIdStub = sinon.stub(getRecordCumulusId, 'getRecordCumulusId').returns(executionCumulusId);
```

Comment:
If you make the change in this function to use execution PG model `getRecordCumulusId` in the function, then you don't need this stub and can pass in a fake model like you are for PDRs and providers

Reply:
[`c58cf1b` (#2051)](https://github.com/nasa/cumulus/pull/2051/commits/c58cf1bbdd841e01b1621ee0b002c6a4c0b8d210)